membership/src/main/kotlin/net/corda/v5/membership/GroupParameters.kt

```diff
@@ -19,23 +19,27 @@ import java.time.Instant
  * int minimumPlatformVersion = groupParameters.getMinimumPlatformVersion();
  * Instant modifiedTime = groupParameters.getModifiedTime();
  * int epoch = groupParameters.getEpoch();
+ * List<NotaryInfo> notaries = groupParameters.getNotaries();
  * ```
  *
  * ```kotlin
  * val groupParameters = fullTransaction.membershipParameters
  * val minimumPlatformVersion = groupParameters?.minimumPlatformVersion
  * val modifiedTime = groupParameters?.modifiedTime
  * val epoch = groupParameters?.epoch
+ * val notaries = groupParameters?.notaries
  * ```
  *
  * @property minimumPlatformVersion The minimum platform version required to be running on in order to transact within a group.
  * @property modifiedTime The [Instant] representing the last time the group parameters were modified.
  * @property epoch An [Int] representing the version of the group parameters. This is incremented on each modification to the
  * group parameters.
+ * @property notaries A list of all available notary services in the group.
  */
 @CordaSerializable
 interface GroupParameters : LayeredPropertyMap {
     val minimumPlatformVersion: Int
     val modifiedTime: Instant
     val epoch: Int
+    val notaries: List<NotaryInfo>
```

Comment:
Do we care about the order? (If not, can we use Collection instead of List)

Reply:
But we will get a list anyway because we have parseList() on layered property map.

Reply:
Or we can use parseSet().

Reply:
This would also mean that i need to modify NotaryLookUp interface to return a collection of notaries or call.toList() on the collection there

Reply:
I think that `parseList` is OK, but it's an implementation detail. We don't want to commit ourselves not to change it in the future unless we have to...

Reply:
OK then changing everything including NotaryLookup to collection